### DISCUSSION TITLE
[Feature Request] Parameter to pin the Export Widget

### DISCUSSION THREAD

**Discussion by Noschvie** [UtilitechAS/amsreader-firmware#656] [Ideas]
Hello
Thanks for this amazing project.

In case of having a PV system the export widget should always be visible. Currently it disapears when a cloud is passing.

![grafik](https://github.com/UtilitechAS/amsreader-firmware/assets/48217146/b9e3cdc6-5e13-4f31-b06e-926f7ea981cd)

![grafik](https://github.com/UtilitechAS/amsreader-firmware/assets/48217146/9e7f6161-d11c-417e-903e-8d736e7fa310)

**Comment by Noschvie** [comment]
In version v2.1.17 it seems to be pinned, isn't it?
[discussions 407](https://github.com/UtilitechAS/amsreader-firmware/discussions/407#discussion-4780149)

**Reply by gskjold** [reply]
Should be pinned if you have configured your peak production in configuration. You can also pin it by setting "Export gauge" under "User interface" to "Show"

**Reply by Noschvie** [reply]
Oh, sorry... changed and works perfect! Thanks
